PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -556,9 +556,9 @@ def miniAOD_customizeCommon(process):
     from Configuration.Eras.Modifier_run2_L1prefiring_cff import run2_L1prefiring
     from Configuration.Eras.Modifier_stage1L1Trigger_cff import stage1L1Trigger
     from Configuration.Eras.Modifier_stage2L1Trigger_2017_cff import stage2L1Trigger_2017
-    process.load("PhysicsTools.PatUtils.L1ECALPrefiringWeightProducer_cff")
-    stage1L1Trigger.toModify(process.prefiringweight, DataEra = "2016BtoH")
-    stage2L1Trigger_2017.toModify(process.prefiringweight, DataEra = "2017BtoF")
+    process.load("PhysicsTools.PatUtils.L1PrefiringWeightProducer_cff")
+    stage1L1Trigger.toModify(process.prefiringweight, DataEraECAL = "2016BtoH", JetMaxMuonFraction = -1, DoMuons = cms.bool(False) )
```

Comment:
It looks like the right modifier for 2016 is stage2L1Trigger, and not stage1L1Trigger. 
https://github.com/cms-sw/cmssw/blob/master/Configuration/Eras/python/Era_Run2_2016_cff.py#L17-L18 
This would explain while failed comparisons in MINIAOD are only seen in 2016.
@JanFSchulte

Reply:
So I guess, the proper line is: 
 `stage2L1Trigger.toModify(process.prefiringweight, DataEraECAL = "2017BtoF", JetMaxMuonFraction = -1, DoMuons = cms.bool(False) )` 
 (note that the 2017 maps must be used since they were apparently changed in the wrong modifier)

Reply:
Thanks for spotting this. I will implement an actually correct fix for the 12X PR